aws/lambda/profile_plots.py

```diff
@@ -0,0 +1,166 @@
+import boto3
+import cmocean
+import json
+import io
+import netCDF4
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import numpy as np
+import numpy.ma as ma
+import matplotlib.cm as cm
+import traceback
+from datetime import datetime
+
+import matplotlib
+matplotlib.use('AGG')
+
+PARAMETERS = {
+    'temperature': {
+        'cmap': cmocean.cm.thermal,
+        'display': u'Temperature (°C)'
+    },
+    'salinity': {
+        'cmap': cmocean.cm.haline,
+        'display': u'Salinity (1e-3)'
+    },
+    'conductivity': {
+        'cmap': cmocean.cm.haline,
+        'display': u'Conductivity (S m-1)'
+    },
+    'density': {
+        'cmap': cmocean.cm.dense,
+        'display': u'Density (kg m-3)',
+    }
+}
+
+
+def lambda_handler(event, context):
+    '''
+    Plot the parameters for a deployment
+    :param dict deployment: A deployment object
+    :param str path: Folder path to where to store the images
+    '''
+    thredds_url = event['dap']
+    deployment_name = thredds_url.split('/')[7]
+
+    with netCDF4.Dataset(thredds_url, 'r') as nc:
+        for parameter in PARAMETERS:
+            filename = '{}/{}.png'.format(deployment_name, parameter)
+            try:
+                plot_from_nc(deployment_name, nc, parameter, filename)
+            except Exception:
+                print("Failed to generate plot for %s" % parameter)
```

Comment:
Use `logger.exception` here instead and send to stdout.
Don't use `print` for anything that should be logged.